src/js/PhotoSphereViewer.public.js

```diff
@@ -206,13 +206,13 @@ PhotoSphereViewer.prototype.setPanorama = function(path, position, transition) {
           this.rotate(position);
         }
       }.bind(this))
-      .ensure(function() {
+      .finally(function() {
         this.loader.hide();
+
         this.canvas_container.style.opacity = 1;
 
         this.prop.loading_promise = null;
-      }.bind(this))
-      .rethrow();
```

Comment:
I had to rethrow because otherwise the Promise chain swallows coding errors like "undefined has not property 'foo'" which makes development /debuging hard.

If this is still the case (probably) we had something else, at least `.catch(err => console.error(err.message))`